### ISSUE TITLE
Stack integration

### ISSUE THREAD

**Issue by ghost** [themattchan/flycheck-liquidhs.el#3]
How do I get the emacs plugin to use "stack exec liquid -- file.hs" rather than "liquid file.hs"?  I need this for liquid to be able to see stack-installed modules.

**Comment by themattchan** [comment]
Right now you can just hack it into the :command string. 

I'm currently trying to add stack support to the checker nicely (i.e. without making two checkers or anything like that). Aiming to have the same checker first try stack then fallback to plain 'ol ghci.

Stay tuned.

**Comment by ghost** [comment]
I have changed the `:command` string in the file on my computer.  In the meantime (before you get the checker working) do you think it is worth giving instructions for what to change in the README?

**Comment by themattchan** [comment]
Yes, indeed. I would think that just changing the :command string would work; I'll add instructions to the readme.
